dev/build/plugin-versions/bignums.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+case "$1" in
+    git-master|/cygdrive*)
+        # First get the SHA of master/HEAD and then download a zip of exactly this state
+        # Note: the access token is a public access token of MSoegtropIMC
```

Comment:
Your are not using this method anymore thus, this comment should be removed or adapted. BTW, I have just checked the various permission levels for personal access token and I don't see any that only gives read access. So I strongly suggest that you still revoke the access token that was included in an earlier version of this PR.

Reply:
Yes you are right, I should remove the comment on the access token and also fix the Readme file.

Regarding access rights of access tokens: when you create an access token the systems asks which rights to give and you simply don't tick any right at all. Github then displays the access rights of this token as "public access" meaning (as far as I can tell) it as the same rights an unauthenticated user has. But I already deleted the token yesterday after switching to SkySkimmer's method, since I create tokens for specific applications and delete them when they are not needed any more.

Reply:
Oh you are right! I had forgotten you could not check any box at all :)